### PR TITLE
Allow CEER efficiency unit for room ac

### LINKED ISSUE #716: Allow CEER rating for window AC
Right now we only support EER. 

CEER (Combined Energy Efficiency Ratio) is the new standard by which Window Air conditioners are rated by the DOE (Department of Energy), as of June 2014. It takes in account the energy used while the air conditioner is running, as well as the standby power used when the unit is not running but is powered on.

This would also need to be added to HPXML (e.g., [here](https://hpxml.nrel.gov/datadictionary/3.0.0/Building/BuildingDetails/Systems/HVAC/HVACPlant/CoolingSystem/AnnualCoolingEfficiency/Units)).

### PR DESCRIPTION
**Pull request by yzhou601**
## Pull Request Description

fix #716 
## Checklist

Not all may apply:

- [x] EPvalidator.xml has been updated
- [x] Tests (and test files) have been updated
- [x] Documentation has been updated
- [x] Changelog has been updated
- [ ] `openstudio tasks.rb update_measures` has been run
- [ ] No unexpected regression test changes on CI (checked comparison artifacts)

### CODE REVIEW

**Comment by yzhou601** [comment]
CI only shows minor diffs on summer peak electricity in test files with room acs. I verified that in room-ac idf, efficiency is decreased by 1.01 by modeling CEER instead of EER. But the total energy diff is just 0.04MBtu, too small to be reflected in comparisons artifacts. This PR is ready for review.

File: HPXMLtoOpenStudio/resources/hpxml.rb [hunk 1]
```diff
@@ -2855,9 +2856,16 @@ def to_oga(doc)
...
       elsif [HVACTypeRoomAirConditioner].include? @cooling_system_type
-        efficiency_units = UnitsEER
-        efficiency_value = @cooling_efficiency_eer
-        efficiency_value_isdefaulted = @cooling_efficiency_eer_isdefaulted
+        if not @cooling_efficiency_eer.nil?
+          efficiency_units = UnitsEER
+          efficiency_value = @cooling_efficiency_eer
+          efficiency_value_isdefaulted = @cooling_efficiency_eer_isdefaulted
+        else
+          # Fixme: In this way if efficiency is specified in EER, CEER will not show up in the xml generated(in.xml), even though in the simulation we're using ceer being converted to.
```
Comment (shorowit) [inline HPXMLtoOpenStudio/resources/hpxml.rb:2864]: Indeed the EER->CEER conversion is not a "default" but rather a correlation we use for modeling. So I suggest:
1. Revert the code in hpxml_defaults.rb and implement the EER->CEER conversion in hvac.rb's apply_room_ac (or whatever it's called) method.
2. Change the code above to `elsif not @cooling_efficiency_ceer.nil?`

We've taken similar approach for water heaters (EF vs UEF) and appliances (EF vs CEF, MEF vs IMEF, etc.).

File: Changelog.md [hunk 2]
```diff
@@ -12,6 +12,7 @@ __New Features__
 - Revises shared mechanical ventilation preconditioning control logic to operate less often.
 - Allows defaulting of HVAC/DHW system efficiency (by age).
 - Allows defaulting of mechanical ventilation fan power (by type).
+- Allows CEER(Combined Energy Efficiency Ratio) efficiency unit for window AC.
```
Comment (shorowit) [inline Changelog.md:15]: Add a space after CEER and change "window" to "room" for consistency.

File: HPXMLtoOpenStudio/resources/hvac.rb [hunk 3]
```diff
@@ -3077,6 +3077,12 @@ def self.set_heat_c_d(heating_system, num_speeds)
     htg_ap.heat_plf_fplr_spec = [calc_plr_coefficients(htg_ap.heat_c_d)] * num_speeds
   end
 
+  def self.set_cool_ceer(cooling_system)
+    return if cooling_system.cooling_system_type != HPXML::HVACTypeRoomAirConditioner
+    # eer must be specified by EPvalidator.
+    cooling_system.cooling_efficiency_ceer = cooling_system.cooling_efficiency_eer / 1.01
```
Comment (shorowit) [inline HPXMLtoOpenStudio/resources/hvac.rb:3083]: Let's add a comment citing the ENERGY STAR document that this regression is based on.

**Review by shorowit — COMMENTED** [review]
This looks great! Nice thorough work. Just a couple small comments.

**Review by shorowit — APPROVED** [review]

**Pull request merged**